### DISCUSSION TITLE
PrismaClientKnownRequestError:

### DISCUSSION THREAD

**Discussion by razsa** [prisma/prisma#14628] [Q&A]
Hello, I was following the Getting Started: Add to existing project guide but when I attempt to query the database I get this error. Any ideas how to sort this out? Thanks.

```
Invalid `prisma.users.findMany()` invocation in
/workspaces/next-auth-example/index.ts:18:41

  15     }
  16 })
  17 
→ 18 const allUsers = await prisma.users.findMany(
  The column `scores.scoreID` does not exist in the current database.
    at RequestHandler.handleRequestError (/workspaces/next-auth-example/node_modules/@prisma/client/runtime/index.js:28658:13)
    at RequestHandler.request (/workspaces/next-auth-example/node_modules/@prisma/client/runtime/index.js:28640:12)
    at async consumer (/workspaces/next-auth-example/node_modules/@prisma/client/runtime/index.js:29618:18)
    at async PrismaClient._request (/workspaces/next-auth-example/node_modules/@prisma/client/runtime/index.js:29639:16) {
  code: 'P2022',
  clientVersion: '4.1.1',
  meta: { column: 'scores.scoreID' }
}
```

My schema:
```prisma
model roleScores {
  roleScoreId    BigInt   @id @default(autoincrement())
  frontend       String?
  backend        String?
  dataScience    String?
  uiux           String?
  projectManager String?
  scores         scores[]
}

model roles {
  roleID   BigInt  @id @default(autoincrement())
  roleName String?
}

model scores {
  scoreID        BigInt      @id @default(autoincrement())
  testVersion    String
  id    BigInt
  roleScoreId BigInt
  users          users       @relation(fields: [id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "scores_consumer_id_83a30005_fk_consumer_ID")
  roleScores    roleScores @relation(fields: [roleScoreId], references: [roleScoreId], onDelete: NoAction, onUpdate: NoAction, map: "scores_rolescoreid_id_5821b72d_fk_role_scores_rolescoreid")

  @@index([id], map: "scores_consumer_id_83a30005")
  @@index([roleScoreId], map: "scores_rolescoreid_id_5821b72d")
}

model users {
  id            BigInt    @id(map: "consumer_pkey") @default(autoincrement())
  firstName     String
  lastName      String
  email         String
  resume        Bytes?
  phoneNumber   String?
  preferredRole String
  otherRoles    String[]
  createAt    DateTime  @default(dbgenerated("CURRENT_DATE")) @db.Date
  updateAt    DateTime? @default(dbgenerated("CURRENT_DATE")) @db.Date
  scores        scores[]
}
```

**Comment by austincrim** [comment]
Hey @razsa 👋,

Have you tried running [`npx prisma db push`](https://www.prisma.io/docs/reference/api-reference/command-reference#db-push) or [`npx prisma migrate dev`](https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-dev) to ensure your database schema accurately reflects the Prisma schema?

**Reply by janpio** [reply]
If I understand @razsa correctly, the schema is the `db pull` output for an existing database? Then this is super weird, as you could clearly talk to that database to introspect the schema, and now it tells you that the column it is trying to use does not exist. Are you sure you are using the same connection string?

**Reply by razsa** [reply]
Connection string?

**Reply by janpio** [reply]
https://www.prisma.io/docs/reference/database-reference/connection-urls

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
